Cmdline/Action/List.cs

```diff
@@ -65,26 +65,49 @@ public int RunCommand(CKAN.KSP ksp, object raw_options)
                     else if (current_version is DllVersion)
                     {
                         // Autodetected dll
-                        bullet = "-";
+                        bullet = "A";
                     }
                     else
                     {
                         try
                         {
                             // Check if upgrades are available, and show appropriately.
                             CkanModule latest = registry.LatestAvailable(mod.Key, ksp.VersionCriteria());
-
+                            CkanModule current = registry.GetInstalledVersion(mod.Key);
+                                
                             log.InfoFormat("Latest {0} is {1}", mod.Key, latest);
 
                             if (latest == null)
                             {
                                 // Not compatible!
                                 bullet = "X";
+                                //Check if mod is replaceable
+                                if ( current.replaced_by != null)
+                                {
+                                    CkanModule replacement = registry.LatestAvailable(current.replaced_by.name, ksp.VersionCriteria());
+                                    if (replacement != null)
+                                    {
+                                        // Replaceable
+                                        bullet = ">";
+                                        modInfo = string.Format("{0} {1} > {2} {3}", mod.Key, mod.Value, replacement.name, replacement.version);
+                                    }
+                                }
                             }
                             else if (latest.version.IsEqualTo(current_version))
                             {
                                 // Up to date
                                 bullet = "-";
+                                //Check if mod is replaceable
+                                if ( current.replaced_by != null)
+                                {
+                                    CkanModule replacement = registry.LatestAvailable(current.replaced_by.name, ksp.VersionCriteria());
+                                    if (replacement != null)
```

Comment:
One of the main questions users seek to answer with CKAN is, "Can I upgrade to the next KSP version yet without losing any of my mods?" Once `replaced_by` is out in the wild, it will become part of that process.

Suppose OriginalMod is for 1.2 only, and RevivalMod replaces it on 1.3 only. Users need to know about that relationship while still on 1.2; is there any place where it's shown currently? It looks like it's not included in `ckan list` so far.

Reply:
The list command as it is doesn't help with that use case at all, since it only lists installed mods and updates that match the current KSP version. To answer that question, we'd need some new functionality based on the "available" command, I think.